app/views/practices/_edit_nav.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class='dm-sidenav practice-edit-nav sticky' style=''>
   <% unless @practice.new_record? %>
-    <div style='padding: .5rem 1rem;'
+    <div style='padding: .5rem 1rem;'>
```

Comment:
Could `class = "padding-y-1 padding-x-2` instead of putting the style directly in the HTML? https://designsystem.digital.gov/utilities/margin-and-padding/

Reply:
I think the better question is, can we remove this file altogether? We haven't used this since I've been on the team as well lol.

Reply:
we're gonna have a story to clean up the practices view directory. it's a mess